src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -95,14 +95,18 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
   const radioStations = useSelector(radioStationsSelector);
   const parentVenue = useSelector(parentVenueSelector);
 
-  const venueParentId = venue?.parentId;
-  const venueParentQuery = useMemo<ReduxFirestoreQuerySetting>(
-    () => ({
-      collection: "venues",
-      doc: venueParentId,
-      storeAs: "parentVenue",
-    }),
-    [venueParentId]
+  const venueParentQuery = useMemo<SparkleRFDocQuery[]>(
```

Comment:
Suggestion: Maybe we could move this query, useConnect and selector into the hook?